mobile/SCREEN_IMPLEMENTATION_PLAN.md

```
@@ -0,0 +1,363 @@
+app/
+├── (tabs)/
+│   ├── _layout.tsx      # Tab navigator config
+│   ├── index.tsx        # Dashboard (home)
+│   ├── campaigns.tsx    # Campaigns list
+│   ├── notes.tsx        # Notes list
+│   ├── npcs.tsx         # NPCs list
+│   └── locations.tsx    # Locations list
+```
+
+**Tab Icons** (MaterialCommunityIcons):
+| Tab | Icon | Active Icon |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)